.github/workflows/util/Makefile

```diff
@@ -0,0 +1,197 @@
+# This Makefile is used to automate Kaili releases.
+# Please, see documentation in RELEASING.adoc
+
+SEMVER ?= './.github/workflows/util/semver'
+
+RELEASE_TYPE ?= 'minor'
+CONSOLE_LOCAL_DIR ?= ./frontend
+
+BACKEND_VERSION ?= $(shell sed -rn 's/^VERSION \?= v(.*)/\1/p' Makefile | xargs)
+
+# Take the version as-is without filtering out the 'v' (if it exists)
+# This will take 'latest' as is or vX.Y.Z-type.
+VERSION_TAG ?= $(shell sed -rn 's/^VERSION \?= (.*)/\1/p' Makefile | xargs)
+
+# Determine what's the version to release, according to the RELEASE_TYPE
+# NOTE: BACKEND_VERSION_TO_RELEASE variable will only be used to update the main Makefile
+# with the version string to publish. Once the main Makefile is updated, all other parts
+# of this ci-Makefile should use BACKEND_VERSION since, then, it will contain the right
+# version string and the value of BACKEND_VERSION_TO_RELEASE will be invalid.
+ifeq ($(RELEASE_TYPE),patch)
+  # For patch, bump the version that is present in Makefile. This will be the
+  # version string to use for the release.
+  BACKEND_VERSION_TO_RELEASE ?= v$(shell $(SEMVER) bump $(RELEASE_TYPE) $(BACKEND_VERSION))
+endif
+ifeq ($(findstring $(RELEASE_TYPE),major minor),$(RELEASE_TYPE))
+  # For major and minor, remove the -SNAPSHOT suffix from the Makefile
+  BACKEND_VERSION_TO_RELEASE ?= v$(shell $(SEMVER) bump release $(BACKEND_VERSION))
+endif
+
+ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = y
+else ifeq ($(RELEASE_TYPE),edge)
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = y
+else
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = n
+endif
+
+ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
+  # For end-of-week snapshot release, substitute the -SNAPSHOT prefix in the Makefile with
+  # the value of $(RELEASE_TYPE) -- assuming value will be snapshot.{X}
+  BACKEND_VERSION_TO_RELEASE ?= v$(shell $(SEMVER) bump prerel $(RELEASE_TYPE) $(BACKEND_VERSION))  
+else ifeq ($(RELEASE_TYPE),edge)
+  # Use 'latest' for edge releases
+  BACKEND_VERSION_TO_RELEASE ?= latest
+endif
+
+ifeq ($(RELEASE_TYPE),edge)
+  # Avoid $(SEMVER) error, as BACKEND_VERSION will return empty string when using 'latest' as the version
+  # By not setting these variable, we avoid a $(SEMVER) error when expanding a rule that uses $(BACKEND_BUMPED_VERSION)
+else
+  # NOTE: The value fo BACKEND_BUMPED_VERSION and BACKEND_VERSION_BRANCH will only be valid
+  # after the main Makefile has been modified with the version being released.
+  BACKEND_BUMPED_VERSION ?= $(shell $(SEMVER) bump minor $(BACKEND_VERSION))
+  BACKEND_VERSION_BRANCH ?= $(shell $(SEMVER) bump release $(BACKEND_VERSION) | sed 's/\.[[:digit:]]\+$$//')
+endif
+
+BACKEND_GITHUB_URI ?= https://github.com/$(BACKEND_REPO).git
+BACKEND_PULL_URI ?= https://api.github.com/repos/$(BACKEND_REPO)/pulls
+BACKEND_RELEASE_URI = $(subst pulls,releases,$(BACKEND_PULL_URI))
+BACKEND_MAIN_BRANCH ?= master
+
+KIALI_BOT_USER ?= leandroberetta
```

Comment:
Also here add a TODO comment.
It's just as a reminder. If you happen to forget it, other eyes can see the TODO comment and fix it for you.